published/private/xdpioctl.h

```diff
@@ -123,4 +123,5 @@ typedef struct _XSK_SET_SOCKOPT_IN {
 typedef struct _XSK_NOTIFY_IN {
     XSK_NOTIFY_FLAGS Flags;
     UINT32 WaitTimeoutMilliseconds;
+    XSK_COMPLETION_CONTEXT CompletionContext;
```

Comment:
hmm, so this would break existing xdpapi.dll with a new xdp.sys? i can't recall if we set an explicit goal that xdpapi.dll itself would be usable with newer versions of xdp.sys.

i guess with our deployment model of xdpapi.dll being packaged directly with xdp.sys, there's not much benefit in supporting mismatched binary versions.

Reply:
Good catch. I can't recall that goal either (we definitely did for native XDP I believe), but I'd rather not be the first to break in this way if we can avoid it. I should re-think this

Reply:
I'm not too worried about some system with one version of the API and another of the driver.

Reply:
yeah, let's state that the IOCTL surface is NOT stable - only the xdpapi.dll export surface is ABI stable